platform/security/handler/security-handler-pki/src/main/java/org/codice/ddf/security/handler/pki/CrlChecker.java

```diff
@@ -78,7 +86,8 @@ private boolean passesCrl(X509Certificate[] certs) {
         if (certs != null) {
             LOGGER.debug("Got {} certificate(s) in the incoming request", certs.length);
             for (X509Certificate curCert : certs) {
-                if (crl.isRevoked(curCert)) {
+                if (crlCache.get()
```

Comment:
The crlCache reference could have been nulled out since the previous check to `crlCache.get()` on line 70. We should not rely on it staying the same for any period of time after it is checked. 
We should either capture the .get90 on line 70 into a variable and pass it as a parameter to passesCrl or we should re fetch here and check again for null.